9.4/scripts.js

```diff
@@ -0,0 +1,12 @@
+
+function drawTree(n) {
+	
+    for (i=0; i<=n; i++) {
+		var star = "";
+        for (j=0; j<=i; j++) {
+            star += '*';
+        }
+        console.log(star);
+    }
+}
+console.log(drawTree(5));
```

Comment:
wyswietla Ci sie `undefined` na koncu choinki poniewaz powinnas tu po prostu wywolac funkcje zamiast przekazywac jej wynik do console.log (console.log masz juz wewnatrz funkcji wiec to wystarczy do wyswtlania gwiazdek).